packages/@aws-cdk/aws-vpclattice-alpha/lib/service.ts

```diff
@@ -0,0 +1,220 @@
+
+  /**
+   * The hosted zone ID of the DNS entry
+   *
+   * @default - No hosted zone ID
+   */
+  readonly hostedZoneId?: string;
+}
+
+/**
+ * Properties for creating a VPC Lattice service
+ */
+export interface ServiceProps {
+  /**
+   * The name of the VPC Lattice service
+   */
+  readonly serviceName: string;
+
+  /**
+   * The authentication type for the service
+   *
+   * @default ServiceAuthType.NONE
+   */
+  readonly authType: ServiceAuthType;
+
+  /**
+   * The authentication policy for the service
+   * Required when authType is AWS_IAM
+   *
+   * @default - No policy
+   */
+  readonly authPolicy: PolicyDocument;
+
+  /**
+   * The certificate to use for the service
+   * Required for HTTPS listeners
+   *
+   * @default - No certificate
+   */
+  readonly certificate?: ICertificate;
+
+  /**
+   * The custom domain name for the service
+   * Required for TLS listeners
+   *
+   * @default - No custom domain name
+   */
+  readonly customDomainName?: string;
+
+  /**
+   * The DNS entry for the service
+   *
+   * @default - No DNS entry
+   */
+  readonly dnsEntry?: DnsEntry;
+}
+
+/**
+ * Interface representing a VPC Lattice service
+ */
+export interface IService extends IResource {
+  /**
+   * The VPC lattice service ARN
+   *
+   * @attribute
+   * @returns a token representing the service ARN
+   */
+  readonly serviceArn: string;
+
+  /**
+   * The VPC lattice service ID
+   *
+   * @attribute
+   * @returns a token representing the service ID
+   */
+  readonly serviceId: string;
+
+  /**
+   * The DNS entry for the VPC lattice service
+   */
+  readonly dnsEntry?: DnsEntry;
+}
+
+/**
+ * Base class for VPC Lattice services
+ */
+abstract class ServiceBase extends Resource implements IService {
+  /**
+   * The VPC lattice service ARN
+   */
+  public abstract readonly serviceArn: string;
+
+  /**
+   * The VPC lattice service ID
+   */
+  public abstract readonly serviceId: string;
+
+  /**
+   * The DNS entry for the VPC lattice service
+   */
+  public readonly dnsEntry?: DnsEntry;
+}
+
+/**
+ * Attributes for importing an existing VPC Lattice service
+ */
+export interface ServiceAttributes {
+  /**
+   * The VPC lattice service ARN
+   */
+  readonly serviceArn: string;
+
+  /**
+   * The VPC lattice service ID
+   */
+  // TODO: restore ID? Constructing the ARN from the ID will need accountId, region, etc
+  // readonly serviceId: string;
+}
+
+/**
+ * A VPC Lattice service
+ */
+export class Service extends ServiceBase {
+  /**
+   * Import an existing VPC Lattice service into this CDK application.
+   *
+   * @param scope the parent creating construct (usually `this`)
+   * @param id the construct's name // TODO
+   * @param attrs attributes of the service
+   * @returns An object representing the imported VPC Lattice service.
+   */
+  public static fromServiceAttributes(scope: Construct, id: string, attrs: ServiceAttributes): IService {
+    class UnownedServiceReference extends ServiceBase {
```

Comment:
Nice! `Unowned` might be to technical, certainly love the `Reference` part. Whatever we land on, we should update the guide with this.